synapse/rest/client/v2_alpha/register.py

```diff
@@ -357,15 +357,9 @@ async def on_GET(self, request):
                 403, "Registration has been disabled", errcode=Codes.FORBIDDEN
             )
 
-        ip = self.hs.get_ip_from_request(request)
-        with self.ratelimiter.ratelimit(ip) as wait_deferred:
```

Comment:
why do you remove the `ratelimiter` check here?

Reply:
Now that this endpoint just always returns `{"available": True}` I don't think there's much point in ratelimiting it.

Reply:
you're right, I agree
I forgot it was change about `/register/available`

Reply:
Yeah, hard to notice in the diff :)